corehq/apps/reports/filters/api.py

```diff
@@ -154,10 +154,36 @@ def extra_data_sources(self):
         # extra data sources to be included for filtering
         raise NotImplementedError('Not implemented yet')
 
+    def custom_locations_search(self):
+        """
+        When the query is specifically searching for just locations and not any other entity like user, group.
+        For ex: enter "/Bihar/patn" would match child locations under locations named Bihar, having name like
+        patn.
+        """
+        return self.q.startswith('/')
+
+    @staticmethod
+    def _get_location_specific_custom_filters(query):
+        query_sections = query.split("/")
+        parent_name = query_sections[1]
+        try:
+            search_query = query_sections[2]
+        except IndexError:
+            search_query = ""
+        return parent_name, search_query
+
     def get_locations_query(self, query):
-        return (SQLLocation.active_objects
-                .filter_path_by_user_input(self.domain, query)
-                .accessible_to_user(self.request.domain, self.request.couch_user))
+        if self.custom_locations_search():
+            parent_name, search_query = self._get_location_specific_custom_filters(query)
+            parents = SQLLocation.active_objects.filter(name=parent_name, domain=self.domain)
```

Comment:
one last thought - you might want this to use `name__iexact` so you don't match case.

Reply:
Do you mean it should match the exact case? Its doing that already, its case sensitive.

Reply:
I was thinking a case insensitive match would be more user friendly, but don't feel strongly about it.

Reply:
I agree with case insensitive search and feel more strongly than Ethan does, both to be more user friendly and to be more consistent with the rest of HQ.

Reply:
Updated [here](https://github.com/dimagi/commcare-hq/pull/17958/commits/13526f9bdc2d646428d700c1d26cc4a1022f8c4a)

Reply:
thanks!